src/keyring.ts

```diff
@@ -137,7 +152,7 @@ export type Keyring<State extends Json> = {
     address: Hex,
     transaction: Transaction,
```

Comment:
we should use the Transaction from @ethereumjs/tx for now?

Reply:
[678ac8c](https://github.com/MetaMask/utils/pull/89/commits/678ac8cd991dfcd9171873ab6d3a3bddb60f585d)